bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -18,49 +18,48 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 <!-- Set default values for grid attributes -->
 <scripgriddata_lrgfile_needed>none</scripgriddata_lrgfile_needed>
-<scripgriddata_type>SCRIP</scripgriddata_type>
```

Comment:
Unnecessary because this information is now picked up from the files on the fly.